### ISSUE TITLE
[BUG]Scaling

### ISSUE THREAD

**Issue by ForBai** [bebeli555/CookieClient#11]
Start minecraft with the client open gui

you see tiny gui my gui is soo tiny i need a monokle to see it

**Comment by bebeli555** [comment]
Ive heard other people have the same issue but its really hard for me to fix it when it works fine for me. Maybe its caused by another client ur running or something. But u can manually scale the gui to be bigger with "++set GUIScale value" command. And the value could be like 0.35 so it would make it bigger

**Comment by ForBai** [comment]
thanks
